master/getting-started/kubernetes/installation/index.md

```diff
@@ -49,23 +49,25 @@ You can find some of them here, organized by cloud provider.
 ## <a id="ipvs-mode"></a>kube-proxy IPVS mode
 
 {{site.prodname}} has beta-level support for `kube-proxy`'s `ipvs` proxy mode.
-{{site.prodname}} `ipvs` support is activated automatically if {{site.prodname}} 
+{{site.prodname}} `ipvs` support is activated automatically if {{site.prodname}}
 detects that `kube-proxy` is running in that mode.
 
-`ipvs` mode promises greater scale and performance vs `iptables` mode.  
+`ipvs` mode promises greater scale and performance vs `iptables` mode.
 However, it comes with some limitations.  In IPVS mode:
 
-- `kube-proxy` has a [known issue](https://github.com/kubernetes/kubernetes/issues/58202) 
-  affecting hosts with host interfaces that that are **not** named using the 
+- `kube-proxy` has a [known issue](https://github.com/kubernetes/kubernetes/issues/58202)
+  affecting hosts with host interfaces that that are **not** named using the
   pattern `ethN`.
-- {{site.prodname}} requires [additional `iptables` packet mark bits](../../../reference/felix/configuration#ipvs-bits) 
+- {{site.prodname}} requires [additional `iptables` packet mark bits](../../../reference/felix/configuration#ipvs-bits)
   in order to track packets as they pass through IPVS.
-- {{site.prodname}} needs to be [configured](../../../reference/felix/configuration)
-  with the port range that is assigned to Kubernetes NodePorts.  If services 
-  do use NodePorts outside {{site.prodname}}'s expected range, 
-  {{site.prodname}} will treat traffic to those ports as host traffic instead 
+- {{site.prodname}} needs to be [configured](../../../reference/felix/configuration#ipvs-portranges)
+  with the port range that is assigned to Kubernetes NodePorts.  If services
+  do use NodePorts outside {{site.prodname}}'s expected range,
+  {{site.prodname}} will treat traffic to those ports as host traffic instead
   of pod traffic.
-- {{site.prodname}} does not yet support Kubernetes Services that make use of a 
+- {{site.prodname}} does not yet support Kubernetes Services that make use of a
   locally-assigned ExternalIP.  {{site.prodname}} does support ExternalIPs that
   are implemented via an external load balancer.
 - {{site.prodname}} has not yet been scale tested with `ipvs`.
+
+ {{site.prodname}} will function normally if you change `kube-proxy`'s proxy mode after {{site.prodname}} has been deployed. Any kubernetes `ipvs` specific configurations need to be [configured](../../../reference/felix/configuration#ipvs-portranges) before `kube-proxy` switching proxy mode from `iptables` to `ipvs`.
```

Comment:
I added a comment here.

Reply:
Rather than saying `will function normally` I think we should say `will detect`.
```
before `kube-proxy` switching proxy mode
``` 
should be 
```
before changing the proxy mode to `ipvs`
```
Probably easiest for me to merge and fix up on my branch...